### DISCUSSION TITLE
Updated documentation for V11?

### DISCUSSION THREAD

**Discussion by JohnKesko** [AvaloniaUI/Avalonia#11446] [General]
Hi, Have been browsing around and there is the discussion on various V11 stuff but in general, will there be an updated documentation for all the new stuff and updated tutorials etc?

**Comment by JohnKesko** [comment]
N/A

**Comment by timunie** [comment]
We're working on it in the background. But it will not be added to the official docs before 11.0 finally is released. In a perfect world, we'd have two docs versions. We try to do it for 12.0, but for 11.0 it doesn't exist.

**Reply by JohnKesko** [reply]
I see, makes sense. Thanks for update!
